libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/maneuver/ManeuverView.java

```diff
@@ -120,11 +123,18 @@ public void setManeuverTypeAndModifier(@NonNull String maneuverType, @Nullable S
    *
    * @param roundaboutAngle angle to be rendered
    */
-  public void setRoundaboutAngle(@FloatRange(from = 60f, to = 300f) float roundaboutAngle) {
-    if (ROUNDABOUT_MANEUVER_TYPES.contains(maneuverType) && this.roundaboutAngle != roundaboutAngle) {
+  public void setRoundaboutAngle(@Nullable @FloatRange(from = 60f, to = 300f) Float roundaboutAngle) {
```

Comment:
Could you add an additional sentence to the Javadoc here, like:
"If the angle value is null, a generic roundabout image will be shown".